database/migrations/2018_07_03_204220_create_default_activity_type_groups_table.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateDefaultActivityTypeGroupsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // create new default activity type groups row
+        Schema::create('default_activity_type_categories', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('translation_key');
+            $table->timestamps();
+        });
+
+        Schema::create('default_activity_types', function ($table) {
+            $table->increments('id');
+            $table->integer('default_activity_type_category_id');
+            $table->string('translation_key');
+            $table->string('location_type');
+            $table->timestamps();
+        });
+
+        // SIMPLE ACTIVITIES
+        DB::table('default_activity_type_categories')->insert([
+            'translation_key' => 'simple_activities',
+            'created_at' => now(),
```

Comment:
Like in old `ActivityTypesTableSeeder` class, you should calculate `now()` only once.